### PR TITLE
add description to readme

### LINKED ISSUE #94: add short description to repo
would be nice to have a one-liner so that this is not empty:

screen shot 2019-02-28 at 11 58 34 am

### PR DESCRIPTION
**Pull request by matthandlersux**
i went to send the repo to a friend and realized it was not entirely self explanatory. i took a stab at adding a description, feel free to suggest any relevant changes.

i also think the one-liner from line 3 can be added as a description to the repo (#94)

resolves #94

### CODE REVIEW

**Comment by matthandlersux** [comment]
@astampoulis @xasopheno thoughts?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
-# `check-protocols`
+# Check Protocols [![CircleCI](https://circleci.com/gh/Originate/check-protocols.svg?style=svg)](https://circleci.com/gh/Originate/check-protocols) [![Waffle.io - Columns and their card count](https://badge.waffle.io/Originate/check-protocols.svg?columns=all)](https://waffle.io/Originate/check-protocols)
```
Comment (soenkehahn) [inline README.md]: Why do you want to add the circleci badge? This should always be green and is a bit redundant.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+
+Test driven development for bash scripts (or any script, really)
```
Comment (soenkehahn) [inline README.md]: The tool is not only for TDD, but for writing tests in general.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+
+## Description
+
+Automated tests help us write well-behaved applications, and that's great, but
+what about all those pesky deploy scripts? How do we test those?
```
Comment (soenkehahn) [inline README.md]: The tool is not just for deploy scripts. So maybe '[...] what about all those pesky little scripts, e.g. deploy scripts? [...]'

File: README.md [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
+
+`check-protocols` is a toolkit for people who wish to apply TDD to bash
```
Comment (soenkehahn) [inline README.md]: `s/toolkit/tool`?

Comment (soenkehahn) [inline README.md]: Should we talk about testing in general, not only TDD?

File: README.md [hunk 5]
```diff
@@ -1,6 +1,17 @@
...
+scripting. Because of its design, `check-protocols` also happens to work for
+*any* script that makes syscalls. It is designed for the ability to be added
```
Comment (soenkehahn) [inline README.md]: Maybe '[...] *any* script, since it mocks out syscalls. [...]'

File: README.md [hunk 6]
```diff
@@ -1,6 +1,17 @@
...
+to existing bash scripts, without the need to alter them.
```
Comment (soenkehahn) [inline README.md]: Remove 'bash'?

File: README.md (hunk 4 above)
Comment (astampoulis) [reply README.md]: :+1:, also it's not only for bash right?

**Comment by astampoulis** [comment]
I like it :+1:

File: README.md (hunk 1 above)
Comment (matthandlersux) [reply README.md]: i think it's a nice way to indicate to people that "we have CI set up for this repo, and we care about the tests passing".

File: README.md [hunk 7]
```diff
@@ -1,6 +1,18 @@
-# `check-protocols`
+# Check Protocols [![CircleCI](https://circleci.com/gh/Originate/check-protocols.svg?style=svg)](https://circleci.com/gh/Originate/check-protocols) [![Waffle.io - Columns and their card count](https://badge.waffle.io/Originate/check-protocols.svg?columns=all)](https://waffle.io/Originate/check-protocols)
+
+Run tests against your scripts without changing your scripts.
```
Comment (matthandlersux) [inline README.md]: @soenkehahn i liked your version better

File: README.md (hunk 1 above)
Comment (soenkehahn) [reply README.md]: Sounds good.

**Pull request merged**